README.rst

```diff
@@ -163,28 +166,73 @@ For this to work you'll need to have Qt5 and Python 3 installed.
 
     brew install https://raw.githubusercontent.com/mu-editor/mu/master/package/extras/qscintilla2.rb
 
-  .. note:: If you have an existing virtual environment it will not have
-     changed to add the new packages. The simplest thing to do is to create a
-     new virtual environment, remembering to use the
-     ``--system-site-packages`` switch so that installed libraries are
-     included. For instance::
+* On Windows::
+
+    Unless you want to build PyQt5 and its dependencies from scratch (a
+    non-trivial task) you'll almost certainly want to use the binaries
+    provided by Riverbank who provide the Python bindings for Qt. However,
+    they've recently switched from providing binary installers to providing
+    binary wheels. But they only provide wheels for the most recent versions
+    of Python. In addition, the MSI installer version interacts awkwardly
+    with virtual environments so you need to install PyQt5 globally and then
+    create the virtual environment with --system-site-packages.
+
+    ============== ============ =========
+    Python Version PyQt Version Installer
+    -------------- ------------ ---------
+    3.4            5.6          .msi
+    -------------- ------------ ---------
+    3.5            5.6          .whl
+    -------------- ------------ ---------
+    3.5            5.7          .whl
+    -------------- ------------ ---------
+    3.5            5.8          .whl
+    -------------- ------------ ---------
+    3.6            5.8          .whl
+    ============== ============ =========
+
+    If you can, use Python 3.5 or greater and install from within the
+    virtualenv using the wheels.
+
+
+.. note:: From this point onwards, the instructions assume that you're
+   using a virtual environment.
+
+   If you have an existing virtual environment it will not have
+   changed to add the new packages. The simplest thing to do is to create a
+   new virtual environment, remembering to use the
+   ``--system-site-packages`` switch so that installed libraries are
+   included. For instance::
 
-        $ virtualenv -p /usr/local/bin/python3 --system-site-packages ~/env/py3
+    $ virtualenv -p /usr/local/bin/python3 --system-site-packages ~/env/py3
 
-     or::
+ or::
 
-        $ mkvirtualenv -p /usr/local/bin/python3 --system-site-packages py3
+    $ mkvirtualenv -p /usr/local/bin/python3 --system-site-packages py3
+
+or::
+
+    c:\dev\mu> py -3.5 -mvenv .venv35
```

Comment:
Also, out of curiosity more than anything, is there a reason to invoke the python module rather than use a similar convention with `mkvirtualenv -p C:\Python35\python.exe`?